### PR TITLE
feat:[ASP 4546 JCM]-Workday - XMatters integration DAG

### PR DESCRIPTION
**Pull request by JCMOSCON1976**
## Description
This PR is to create a DAG to run a script that synchronizes employee data from Workday to the XMatters system.
This DAG requires the creation of an Airflow Jira connection named eam_jira_connection_id.

## Related Tickets & Documents
JIRA TICKET [ASP 4546 ](https://mozilla-hub.atlassian.net/browse/ASP-4546)
JIRA EPIC TICKET [ASP-4509](https://mozilla-hub.atlassian.net/browse/ASP-4509)

[ASP-4509]: https://mozilla-hub.atlassian.net/browse/ASP-4509?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by spatilmoz** [comment]
Hi @JCMOSCON1976 ,
looks good to me. just one thing - can we make the exception message more non technical user friendly?

**Review by spatilmoz — COMMENTED** [review]
looks good to me. just one thing - can we make the exception message more non technical user friendly?

File: dags/eam_workday_xmatters_integration.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        # use this link to find project id and issue type ids :
+        # https://mozilla-hub.atlassian.net/rest/api/latest/project/ASP
+        project_id=10051,
+        issue_type_id=10007,
+        labels=["airflow-task-failure"],
+    ).notify(context)
+
+
+default_args = {
+    "owner": "Julio Moscon",
```
Comment (scholtzan) [inline dags/eam_workday_xmatters_integration.py:42]:
```suggestion
    "owner": "jmoscon@mozilla.com",
```

**Review by scholtzan — APPROVED** [review]
r+wc

**Comment by badboy** [comment]
sorry to jump in from the side, but does this require the code from https://github.com/mozilla/docker-etl/pull/199 to actually land?

**Pull request merged**